src/main/java/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -648,12 +651,127 @@ public Chat getChat() {
   }
 
   public static void main(final String[] args) {
-    GameRunner.handleCommandLineArgs(args, getProperties(), GameRunner.GameMode.HEADLESS_BOT);
+    System.getProperties().setProperty(GameRunner.TRIPLEA_HEADLESS, "true");
```

Comment:
This property used to be buried a bit in the handle command line args. Noting that it is moved to be the first thing done.